test/unit/index.js

```diff
@@ -1,5 +1,11 @@
 /** @jsx React.DOM */
 
+// **********************
+// Dependencies
+// **********************
+require('keen-js');
+window.$ = window.jQuery = require('jquery-browserify');
```

Comment:
is this line needed? The tests seem to work with just `require('jquery-browserify')`.